tests/python/gaia-ui-tests/gaiatest/tests/functional/settings/test_settings_wifi.py

```diff
@@ -20,7 +20,9 @@ def test_connect_to_wifi_via_settings_app(self):
         settings.launch()
         wifi_settings = settings.open_wifi()
 
+        self.assertFalse(wifi_settings.is_wifi_enabled, "WiFi should be disabled")
```

Comment:
No problem. I meant: If the test fails at that point, we'll see in the stacktrace: 
`self.assertFalse(wifi_settings.is_wifi_enabled, ...)` => the same information will be displayed. In other words, the comment saying "wifi should be disabled" doesn't add any extra information. Hence, that's a kind of duplication in our code base. That's why I think it's better if we remove them.